ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -604,6 +606,10 @@ module Vendor_amd = struct
                       sched;
                       framebufferbytes = mib framebuffer_sz;
                     });
+                  experimental =
```

Comment:
I feel this would be readable and succinct (one line instead of four!) as
`experimental = (experimental <> '0')`
(where the parentheses are just to aid readability).